frontend-react/src/components/Crumbs.tsx

```diff
@@ -11,28 +16,49 @@ interface CrumbConfig {
 interface CrumbsProps {
     crumbList?: CrumbConfig[];
     noPadding?: boolean;
+    previousPage?: string;
 }
 
-const Crumbs = ({ crumbList, noPadding }: CrumbsProps) => {
-    if (crumbList) {
+const Crumbs = ({ crumbList, noPadding, previousPage }: CrumbsProps) => {
+    if (crumbList || previousPage) {
         return (
             <div className={!noPadding ? "grid-container" : ""}>
-                <BreadcrumbBar>
-                    {crumbList?.map((crumbConfig) => (
-                        <Breadcrumb
-                            key={`${crumbConfig.label}`}
-                            current={!crumbConfig?.path}
+                {previousPage !== undefined ? (
+                    <div className="font-sans-lg line-height-sans-4 margin-top-8">
+                        <IconButton
+                            size="big"
+                            type="button"
+                            onClick={() => window.history.back()}
                         >
-                            {crumbConfig?.path ? (
-                                <USCrumbLink href={crumbConfig?.path || ""}>
-                                    {crumbConfig.label}
-                                </USCrumbLink>
-                            ) : (
-                                crumbConfig.label
-                            )}
-                        </Breadcrumb>
-                    ))}
-                </BreadcrumbBar>
+                            <IconArrowBack />
+                        </IconButton>
+                        Return to {previousPage}
+                    </div>
+                ) : null}
+                {crumbList !== undefined && crumbList.length > 0 ? (
```

Comment:
This can be shortened to `crumbList?.length > 0` 🧹

Reply:
Unfortunately typescript will still assume crumbList is possibly undefined later without an explicit undefined check. I presume this is possibly why your Boolean() suggestion in another comment isn't allowed.

Reply:
Oh, TypeScript why ya dumb!  That's the entire point of safe navigation 🤦 

No worries -- carry on then!